core/kazoo_documents/src/kzd_app.erl

```diff
@@ -225,18 +225,26 @@ account_id(JObj) ->
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
--spec users(kz_json:object()) -> api_binaries().
+-spec users(kz_json:object()) -> api_objects().
```

Comment:
@fenollp isn't users a list of Ids ? `is_authorized` in cb_apps_util does a `lists:member(UserId, UserIds);`